src/styles/globals.css

```diff
@@ -1,3 +1,80 @@
+:root {
```

Comment:
Same comment re: the extensive styling you have done globally here. This will change once we adopt a design.

Reply:
Leave it for now; the engineers working on CSS will modify it